examples/connectivity/plot_mne_inverse_label_connectivity.py

```diff
@@ -106,7 +107,7 @@
     label_ypos.append(ypos)
 
 # Reorder the labels based on their location
-lh_labels = [label for (ypos, label) in sorted(zip(label_ypos, lh_labels))]
+lh_labels = [label for (yp, label) in sorted(zip(label_ypos, lh_labels))]
```

Comment:
No it's not used afterward in the way you think. It's one of these sorts of things (F812):
````
data = []
x = 0
a = [x for x in data]  # F812 flake error
```
If anything this change could prevented a future stupid error, but as it stands it does nothing other than avoid a flake error.